<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectDialog</class>
 <widget class="QDialog" name="ConnectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>558</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Little Navmap - Connect</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidgetConnect">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabConnectFsx">
      <attribute name="title">
       <string>&amp;FSX, Prepar3D or MSFS</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="4" column="0" colspan="2">
        <widget class="QCheckBox" name="checkBoxConnectFetchAiShipFsx">
         <property name="toolTip">
          <string>Deselect to disable fetching and transmission of AI or multiplayer ships</string>
         </property>
         <property name="text">
          <string>Fetch AI or multiplayer &amp;ships</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QCheckBox" name="checkBoxConnectFetchAiAircraftFsx">
         <property name="toolTip">
          <string>Deselect to disable fetching and transmission of AI or multiplayer aircraft</string>
         </property>
         <property name="text">
          <string>Fetch AI or multiplayer &amp;aircraft</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="spinBoxConnectAiFetchRadius">
         <property name="toolTip">
          <string>Changes the radius around the user aircraft that is used to include AI or multiplayer aircraft.
Using higher values with a lot of AI or multiplayer aircraft might induce stutters in the simulator.
The unit is always nautical miles (1 NM equals to 1.852 km).
FSX, Prepar3D and MSFS allow a maximum of 105 NM.</string>
         </property>
         <property name="suffix">
          <string> NM</string>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>105</number>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
         <property name="value">
          <number>105</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labelConnectAiFetchRadius">
         <property name="text">
          <string>&amp;AI fetch radius:</string>
         </property>
         <property name="buddy">
          <cstring>spinBoxConnectAiFetchRadius</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelConnectUpdateRateFsx">
         <property name="text">
          <string>&amp;Update time interval:</string>
         </property>
         <property name="buddy">
          <cstring>spinBoxConnectUpdateRateFsx</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="spinBoxConnectUpdateRateFsx">
         <property name="toolTip">
          <string>Allowed range is 50 up to 5000 milliseconds (5 seconds).
Little Navmap fetches data from the flight simulator using this time interval.
Lower values will result in a more fluid aircraft movement but can potentially induce stutters in the simulator.</string>
         </property>
         <property name="suffix">
          <string> ms</string>
         </property>
         <property name="minimum">
          <number>50</number>
         </property>
         <property name="maximum">
          <number>5000</number>
         </property>
         <property name="singleStep">
          <number>50</number>
         </property>
         <property name="value">
          <number>500</number>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="labelConnectFsx">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="text">
          <string>Connect directly to FSX, Prepar3D or MSFS running on the same computer as &lt;i&gt;Little Navmap&lt;/i&gt;</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <property name="margin">
          <number>5</number>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabConnectXp">
      <attribute name="title">
       <string>&amp;X-Plane</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="2" column="0" colspan="2">
        <widget class="QCheckBox" name="checkBoxConnectFetchAiAircraftXp">
         <property name="toolTip">
          <string>Deselect to disable fetching and transmission of AI or multiplayer aircraft</string>
         </property>
         <property name="text">
          <string>Fetch AI or multiplayer &amp;aircraft</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="2">
        <widget class="QLabel" name="labelOptionsUpdatesHint">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/littlenavmap/resources/icons/bulb.svg&quot;/&gt;&lt;span style=&quot; font-style:italic;&quot;&gt; You will need to install the Little Xpconnect X-Plane plugin to connect to the simulator. Go to the main menu -&amp;gt; &amp;quot;Tools&amp;quot; -&amp;gt; &amp;quot;Install Little Xpconnect in X-Plane Plugins ...&amp;quot; to let Little Navmap do this for you.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <property name="margin">
          <number>10</number>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="spinBoxConnectUpdateRateXp">
         <property name="toolTip">
          <string>Allowed range is 50 up to 5000 milliseconds (5 seconds).
Little Navmap fetches data from the flight simulator using this time interval.
Lower values will result in a more fluid aircraft movement but can potentially induce stutters in the simulator.</string>
         </property>
         <property name="suffix">
          <string> ms</string>
         </property>
         <property name="minimum">
          <number>50</number>
         </property>
         <property name="maximum">
          <number>5000</number>
         </property>
         <property name="singleStep">
          <number>50</number>
         </property>
         <property name="value">
          <number>500</number>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="labelConnectXp">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="text">
          <string>Connect directly to X-Plane running on the same computer as &lt;i&gt;Little Navmap&lt;/i&gt;</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <property name="margin">
          <number>5</number>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelConnectUpdateRateXp">
         <property name="text">
          <string>&amp;Update time interval:</string>
         </property>
         <property name="buddy">
          <cstring>spinBoxConnectUpdateRateXp</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QCheckBox" name="checkBoxConnectFetchAiShipXp">
         <property name="toolTip">
          <string>Deselect to disable fetching and transmission of AI or multiplayer ships</string>
         </property>
         <property name="text">
          <string>Fetch AI or multiplayer &amp;ships</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabConnectRemote">
      <attribute name="title">
       <string>&amp;Remote / Network</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="1" column="0">
        <widget class="QLabel" name="labelConnectHostname">
         <property name="text">
          <string>&amp;Hostname or IP address:</string>
         </property>
         <property name="buddy">
          <cstring>comboBoxConnectHostname</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="2">
        <widget class="QSpinBox" name="spinBoxConnectPort">
         <property name="toolTip">
          <string>Add the &lt;span style=&quot;color: #ff0000; font-weight:bold&quot;&gt;port&lt;/span&gt; that Little Navconnect shows on startup.
Use your X-Plane computer name if you are running the Little Xpconnect X-Plane plugin.</string>
         </property>
         <property name="minimum">
          <number>1024</number>
         </property>
         <property name="maximum">
          <number>65535</number>
         </property>
         <property name="value">
          <number>51968</number>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="2">
        <widget class="QComboBox" name="comboBoxConnectHostname">
         <property name="toolTip">
          <string>Add the &lt;span style=&quot;color: #0000ff; font-weight:bold&quot;&gt;hostname&lt;/span&gt; or &lt;span style=&quot;color: #0000ff; font-weight:bold&quot;&gt;IP address&lt;/span&gt; that Little Navconnect shows on startup.
Use your X-Plane computer name if you are running the Little Xpconnect X-Plane plugin.</string>
         </property>
         <property name="editable">
          <bool>true</bool>
         </property>
         <property name="currentText">
          <string/>
         </property>
         <property name="maxCount">
          <number>10</number>
         </property>
         <property name="insertPolicy">
          <enum>QComboBox::InsertAtTop</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QPushButton" name="pushButtonConnectDeleteHostname">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Delete the current hostname or IP address from the list.</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../littlenavmap.qrc">
           <normaloff>:/littlenavmap/resources/icons/cancel.svg</normaloff>:/littlenavmap/resources/icons/cancel.svg</iconset>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="4">
        <widget class="QLabel" name="labelConnectRemote">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="text">
          <string>Connect to a flight simulator using &lt;i&gt;Little Navconnect&lt;/i&gt; where &lt;i&gt;Little Navconnect&lt;/i&gt; and the simulator are running on the flying computer</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <property name="margin">
          <number>5</number>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labelConnectPort">
         <property name="text">
          <string>&amp;Port:</string>
         </property>
         <property name="buddy">
          <cstring>spinBoxConnectPort</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelConnectWarning">
     <property name="text">
      <string/>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxConnectOnStartup">
     <property name="toolTip">
      <string>Automatically connect to given address and port of a remote simulator or to the local flight simulator.
You have to click &quot;Connect&quot; to start the constant connection checking.
Deselecting this will stop all automatic connection attempts immediately.</string>
     </property>
     <property name="text">
      <string>Connect &amp;automatically</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignBottom">
    <widget class="QDialogButtonBox" name="buttonBoxConnect">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close|QDialogButtonBox::Help|QDialogButtonBox::Ok|QDialogButtonBox::Reset</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
  <zorder>checkBoxConnectOnStartup</zorder>
  <zorder>buttonBoxConnect</zorder>
  <zorder>tabWidgetConnect</zorder>
  <zorder>labelConnectWarning</zorder>
 </widget>
 <tabstops>
  <tabstop>tabWidgetConnect</tabstop>
  <tabstop>spinBoxConnectUpdateRateFsx</tabstop>
  <tabstop>spinBoxConnectAiFetchRadius</tabstop>
  <tabstop>checkBoxConnectFetchAiAircraftFsx</tabstop>
  <tabstop>checkBoxConnectFetchAiShipFsx</tabstop>
  <tabstop>checkBoxConnectOnStartup</tabstop>
  <tabstop>spinBoxConnectUpdateRateXp</tabstop>
  <tabstop>checkBoxConnectFetchAiAircraftXp</tabstop>
  <tabstop>checkBoxConnectFetchAiShipXp</tabstop>
  <tabstop>comboBoxConnectHostname</tabstop>
  <tabstop>pushButtonConnectDeleteHostname</tabstop>
  <tabstop>spinBoxConnectPort</tabstop>
 </tabstops>
 <resources>
  <include location="../../littlenavmap.qrc"/>
 </resources>
 <connections/>
</ui>
